tools-v2/README.md

```diff
@@ -2017,6 +2019,66 @@ Output:
 | 0171a33b-17b7-4215-9f00-6d8de2686f77 | testsnap1    | success |
 +--------------------------------------+--------------+---------+
 ```
+### export 
+
+#### export target 
+
+export services and clients in the bs cluster
+
+Usage:
+
```

Comment:
Add specific commands.
curve export or curve export target?

Reply:
My mistake, have fixed it